README.md

```diff
@@ -1,89 +1,75 @@
+[![Github All Releases](https://img.shields.io/github/downloads/Flow-Launcher/Flow.Launcher/total.svg)](https://github.com/Flow-Launcher/Flow.Launcher/releases)
+[![GitHub release (latest by date)](https://img.shields.io/github/v/release/Flow-Launcher/Flow.Launcher)](https://github.com/Flow-Launcher/Flow.Launcher/releases/latest)
+![GitHub Release Date](https://img.shields.io/github/release-date/Flow-Launcher/Flow.Launcher)
+[![Build Status](https://dev.azure.com/Flow-Launcher/Flow.Launcher/_apis/build/status/Flow.Launcher?branchName=master)](https://dev.azure.com/Flow-Launcher/Flow.Launcher/_build/latest?definitionId=1&branchName=master)
 
-**WoX** is a launcher for Windows that simply works. It's an alternative to [Alfred](https://www.alfredapp.com/) and [Launchy](http://www.launchy.net/). You can call it Windows omni-eXecutor if you want a long name.
+Nickname flow, full name Flow Launcher. Dedicated to make your workflow flow more seamlessly. Aimed at being more than an app launcher, it searches, integrates and expands on functionalities. Flow will continue to evolve, designed to be open and built with the community at heart.
 
-![demo](http://i.imgur.com/DtxNBJi.gif)
+<sub>PS. Flowy is its street name.</sub>
 
-Features
---------
+## Status
 
-- Search for everything—applications, **uwp**, folders, files and more.
-- Use *pinyin* to search for programs / 支持用 **拼音** 搜索程序
-  - wyy / wangyiyun → 网易云音乐
-- Keyword plugin search `g search_term`
-- Search youtube, google, twitter and many more
-- Build custom themes at http://www.wox.one/theme/builder
-- Install plugins from http://www.wox.one/plugin
+Flow is under heavy development, but the code base is stable, so contributions are very welcome. If you would like to help maintain it, please do not hesistate to get in touch.
 
-**New from this fork:**
-- Upgraded to .Net Core 3.1
-- Portable mode
-- Drastically improved search experience
-- Auto-complete text suggestion
-- Search all subfolders and files
-- Option to always run CMD or Powershell as administrator
-- Run CMD, Powershell and programs as a different user
-- Manage what programs should be loaded
-- Highlighting of how results are matched during query search
-- Open web search result as a tab or a new window
-- Automatic update
-- Reload/update plugin data
+## Features
 
-Installation
-------------
+- Search everything from applications, folders, bookmarks, YouTube, Twitter and more. All from the comfort of your keyboard without ever touching the mouse.
+- Run batch and PowerShell commands as Administrator or a different user.
+- Support languages from Chinese to Italian and more.
+- Support of wide range of plugins.
+- Fully portable.
 
-View new features released from this fork since Flow.Launcher v1.3.524: [new releases](https://github.com/jjw24/Flow.Launcher/releases)
+## Running Flow Launcher
+Remember to star it, flow will love you more :)
```

Comment:
I'd remove this one 

```suggestion
```

Reply:
Don't want to remind users? Thought it's a good idea

Reply:
Yeah, but perhaps near the top or the bottom, writing this under the Running section feels weird

Reply:
What about this at the end of the Running section:
"Remember to start it, you will get more Chi from Flow :)"

Reply:
I liked the original message, but I'd put it right before the `features` section. I think that's a nice place 😄

Reply:
Done :)